tests/metadata/test_metadata_service.py

```diff
@@ -34,9 +38,9 @@ def setUp(self):
 
     def test_initialization(self):
         """Test that the metadata service registers its handlers correctly"""
-        # Verify that the correct request handler was set up via the call to register during test setup
-        self.service_provider.server.set_request_handler.assert_called_once_with(
-            METADATA_LIST_REQUEST, self.metadata_service._handle_metadata_list_request)
```

Comment:
Do we not still need METADATA_LIST_REQUEST, self.metadata_service._handle_metadata_list_request for test_metadata_list_request?